lib/manageiq/providers/azure_stack/engine.rb

```diff
@@ -4,22 +4,7 @@ module AzureStack
       class Engine < ::Rails::Engine
         isolate_namespace ManageIQ::Providers::AzureStack
 
-        # NOTE:  If you are going to make changes to autoload_paths, please make
-        # sure they are all strings.  Rails will push these paths into the
-        # $LOAD_PATH.
-        #
-        # More info can be found in the ruby-lang bug:
-        #
-        #   https://bugs.ruby-lang.org/issues/14372
-        #
-        # If you have no need to add any autoload_paths in this manageiq
-        # plugin, feel free to delete this section.
-        #
-        # Examples:
-        #
-        # config.autoload_paths << root.join("app", "models").to_s
-        # config.autoload_paths << root.join("lib", "my_provider").to_s
-        # config.autoload_paths << Rails.root.join("app", "models", "aliases").to_s
```

Comment:
I removed all 3 of these, because 2 don't exist, and app/models is automatically on the autoload_paths.